frontend/scripts/react-components/shared/profile-selector/profile-panel/profile-step-panel/profiles-sources-panel.component.jsx

```diff
@@ -10,7 +10,7 @@ import Accordion from 'react-components/shared/accordion/accordion.component';
 import ResizeListener from 'react-components/shared/resize-listener.component';
 import { BREAKPOINTS } from 'constants';
 
-import 'react-components/dashboard-element/dashboard-panel/sources-panel.scss';
+// import 'react-components/nodes-panel/country-sources-panel/sources-panel.scss';
```

Comment:
Why not just removing it?

Reply:
we need some styles will make a separate file
thanks